src/averageAges.js

```diff
@@ -55,7 +76,20 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  let children = people.filter(
+    person => people.find(mother => mother.name === person.mother));
+
+  if (onlyWithSon) {
+    children = children.filter(child => child.sex === 'm');
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/206905904-397d999f-9592-4d02-878a-0d4ad1670cdd.png)

Reply:
Fix it everywhere